variation-commons-core/src/main/java/uk/ac/ebi/eva/commons/core/models/AbstractVariant.java

```diff
@@ -141,7 +141,10 @@ public int getLength() {
     public VariantType getType() {
         if (this.alternate.equals(".")) {
             return VariantType.NO_ALTERNATE;
-        } else if (reference.length() == alternate.length()) {
+        } else if (this.reference.startsWith("(") && this.reference.endsWith(")")
```

Comment:
Should we create these objects using the (legacy) dbSNP input or angle-bracketed symbolic alleles?

Reply:
my reasoning was that writing named variants as symbolic alleles is just a "formatting" decision. the variant model (in the (mongo) DB) is stored with parenthesis. In eva accession this was followed: in the ContextNucleotideAdditionProcessor processor, the alleles are kept with parenthesis (also, named alleles are always moved to ALT column). only later in  VariantToVariantContextProcessor the parenthesis are replaced by angle brackets.

I'm ok saying "SEQUENCE_ALTERATION if contains angle brackets" but then in any place we read variants from the (mongo) DB we have to replace the parenthesis before anything else.